sparse/tests/test_coo.py

```diff
@@ -1745,3 +1745,52 @@ def test_squeeze_validation(self):
 
         with pytest.raises(ValueError, match="Specified axis `0` has a size greater than one: 3"):
             s_arr.squeeze(0)
+
+
+class TestUnique:
```

Comment:
I'd parametrize the test over fill-values as well.

Reply:
Done!